### PR TITLE
Release automation

### PR DESCRIPTION
**Pull request by fedepaol**
- Add a CI step to create the helm chart
- Add a makefile rule to automate the release process (with versioning the tags around the manifests)
- Create an empty release notes file

### CODE REVIEW

File: HOW_TO_RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+git push
+```
+
+## Finalize the release notes
+
+All release notes are always written on the main branch in the `RELEASE_NOTES.md` file, and copied into release branches in a later step. Point out all new features and actions required by users. If there are very notable bugfixes (e.g. security issues, long-term pain point resolved), point those out as well.
+
+For each new release, a section like `## Release vX.Y.Z` must be added.
+
+To get a list of contributors to the release, run git log --format="%aN" $(git merge-base CUR-BRANCH PREV-TAG)..CUR-BRANCH | sort -u | tr '\n' ',' | sed -e 's/,/, /g'. CUR-BRANCH is main if you’re making a minor release (e.g. 0.9.0), or the release branch for the current version if you’re making a patch release (e.g. v0.8 if you’re making v0.8.4). PREV-TAG is the release tag name for the last release (e.g. if you’re preparing 0.8.4, PREV-TAG is v0.8.3. Also think about whether there were significant contributions that weren’t in the form of a commit, and include those people as well. It’s better to err on the side of being too thankful!
```
Comment (oribon) [inline HOW_TO_RELEASE.md:19]: nit: wrap the git log command in ``

File: HOW_TO_RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Run `FRRK8S_VERSION="X.Y.Z" make cutrelease` from the main branch. This will create the appropriate branches, commits and tags in your local repository.
+
+## Push the new artifacts
+Run git push --tags origin main `vX.Y`. This will push all pending changes both in main and the release branch, as well as the new tag for the release.
+
+## Create a new release on github
+By default, new tags show up de-emphasized in the list of releases. Create a new release attached to the tag you just pushed. Make the description point to the release notes on the website.
+
+## Wait for the image repositories to update
+When you pushed, GitHub actions kicked off a set of image builds for the new tag. You need to wait for these images to be pushed live before continuing, because the manifests for the new release point to image tags that don’t exist until GH actions makes them exist.
```
Comment (oribon) [inline HOW_TO_RELEASE.md:39]: before continuing to?

File: HOW_TO_RELEASE.md (hunk 1 above)
Comment (fedepaol) [reply HOW_TO_RELEASE.md:19]: done

File: HOW_TO_RELEASE.md (hunk 2 above)
Comment (fedepaol) [reply HOW_TO_RELEASE.md:39]: reprhased

**Review by oribon — APPROVED** [review]
lgtm

**Pull request merged**